projects/netgrif-components-core/src/lib/search/models/predicate/editable-clause-predicate.ts

```diff
@@ -35,10 +35,6 @@ export class EditableClausePredicate extends EditablePredicate implements OnDest
         });
     }
 
-    ngOnDestroy(): void {
-        this._childUpdated$.complete();
```

Comment:
with ivy it is not possible using angular lifecycles if class isn't part of angular components, so it was removed

Reply:
should we find another way for closing these streams? This might create memory leaks when dealing with search GUI. It should be possible to do, since these classes are almost exclusively bound to components in the search GUI, so those components might call some destroy method

Reply:
its resolved, its called destroy from child class